cuda/syncPrims/denovo/Makefile

```diff
@@ -1,4 +1,4 @@
-CUDA_DIR = /usr/local/cuda
+CUDA_DIR ?= /usr/local/cuda
```

Comment:
I suspect you aren't using this version of the code, but the change is fine more broadly.